src/alexandria3k/__main__.py

```diff
@@ -157,7 +157,7 @@ def add_subcommand_populate(subparsers):
         "-s",
         "--sample",
         # By default the function always returns True
-        default="True",
+        default=("True", "True"),
```

Comment:
The default value must be a string, because it is passed to eval, right?

Reply:
You want something that will work as intended (and not almost by accident 😃) in the line `    sample = eval(f"lambda path: {args.sample}").  What you wrote results in sample returning the **string** `True`, which is also truish in Python, but not what you seem to be intending.  So use a proper string of a tuple of function results `(True, True)`, which is close to what the user would pass.